packages/odyssey-react/src/components/Text/Text.module.scss

```diff
@@ -11,17 +11,25 @@
  */
 
 .root {
+  color: var(--ColorTextBody);
+  font-family: var(--FontFamilyBase);
+  font-size: var(--FontSizeBase);
+  font-variant: normal;
+  letter-spacing: 0;
+  font-feature-settings: "lnum", "pnum";
+  text-decoration-skip-ink: all;
+
   // stylelint-disable-next-line selector-max-type
   abbr,
   &.abbr {
-    border-block-end: 1px dashed $color-primary-dark;
+    border-block-end: 1px dashed var(--ColorBorderBlockEndAbbr);
```

Comment:
I can imagine folks would want to be able to theme width and style as well

Reply:
let me add these good catch